include/mui/TextEditor_mcc.h

```diff
@@ -109,6 +109,8 @@ extern "C" {
 #define MUIA_TextEditor_MatchedKeyword        (TextEditor_Dummy + 0x41)
 #define MUIA_TextEditor_CursorIndex           (TextEditor_Dummy + 0x42)
 #define MUIA_TextEditor_RGBMode               (TextEditor_Dummy + 0x45)
+#define MUIA_TextEditor_HorizontalScrolling   (TextEditor_Dummy + 0x2d)
```

Comment:
After thinking about these two new attributes again please do:

1. Remove `MUIA_TextEditor_HorizontalScrolling` completly and just set FLG_HScroll if `MUIA_TextEditor_HorizontalScollbar` is set to non-NULL. Otherwise remove the flag. Thus, by passing a NULL pointer to that attribute it will automatically disable Horizontal scrolling. So there is no need for another BOOLEAN attribute like that one.
2. Rename `MUIA_TextEditor_HorizontalScrollbar` to `MUIA_TextEditor_HorizontalSlider` because TextEditor already uses the term "Slider" for the vertical slider gadget